src/app/beer_garden/api/http/authentication/__init__.py

```diff
@@ -157,14 +234,3 @@ def _get_access_token_expiration() -> datetime:
 def _get_refresh_token_expiration() -> datetime:
     """Calculate and return the refresh token expiration time"""
     return datetime.utcnow() + timedelta(hours=12)
-
-
-def _decode_token(token) -> dict:
```

Comment:
This got refactored into `decode_token` above with necessary changes to make it usable with the `EventSocket` handler.